torch/library.h

```diff
@@ -543,7 +537,7 @@ class TORCH_API Library final {
   }
   template <typename Func>
   Library& impl_UNBOXED(detail::SelectiveStr<true> name, Func* raw_f) & {
-    return impl(name.operator const char*(), CppFunction::makeUnboxedOnly(raw_f));
+    static_assert(c10::guts::false_t<Func>(), ".impl_UNBOXED(...) was removed. Please use .impl(...) instead.");
```

Comment:
FYI looks like the lack of return statement here is the cause of some/all of your build errors